samples/javascript_nodejs/07.using-adaptive-cards/index.js

```diff
@@ -49,11 +49,11 @@ const adapter = new BotFrameworkAdapter({
 });
 
 // Catch-all for errors.
-adapter.onTurnError = async (turnContext, error) => {
+adapter.onTurnError = async (context, error) => {
     // This check writes out errors to console log v.s. Application Insights.
     console.error(`\n [onTurnError]: ${ error }`);
     // Send a message to the user.
-    await turnContext.sendActivity(`Oops. Something went wrong!`);
+    await context.sendActivity(`Oops. Something went wrong!`);
     // Clear out state
     await conversationState.load(context);
```

Comment:
This sample doesn't use any state, the `conversationState` logic can be removed.